src/main/java/org/maxgamer/quickshop/integration/IntegrationHelper.java

```diff
@@ -72,7 +73,9 @@ public void searchAndRegisterPlugins() {
         PluginManager pluginManager = plugin.getServer().getPluginManager();
         for (Map.Entry<String, Class<? extends IntegratedPlugin>> entry : integratedPluginNameMap.entrySet()) {
             String pluginName = entry.getKey();
-            if (pluginManager.isPluginEnabled(pluginName) && plugin.getConfig().getBoolean("integration." + pluginName.toLowerCase() + ".enable")) {
+            if (pluginName.equalsIgnoreCase("FabledSkyblock") && plugin.getConfig().getBoolean("integration." + pluginName.toLowerCase() + ".enable")) {
+                register(entry.getValue());
```

Comment:
A more ideal way is just add soft-depend in plugin.yml, so the plugin will enabled before quickshop enable.

Reply:
its fine